artemis-chart/values.yaml

```diff
@@ -45,6 +45,7 @@ limitRequests: 20
 captchaWindow: 900000
 captchaTries: 4
 artemisWebBaseDir: ""
+serviceAccountPath: ""
```

Comment:
shouldn't also `.env` and  the  docker-compose files where this env var is passed be updated?